dashboard/common/db_util.py

```diff
@@ -57,11 +57,12 @@ def get_course_view_options (course_id):
     course_view_option = ""
     if (course_id):
         with django.db.connection.cursor() as cursor:
-            cursor.execute("SELECT show_resources_accessed, show_assignment_planning, show_grade_distribution FROM course_view_option WHERE course_id = %s", [course_id])
+            cursor.execute("SELECT show_resources_accessed, show_assignment_planning_v1, show_assignment_planning, show_grade_distribution FROM course_view_option WHERE course_id = %s", [course_id])
             row = cursor.fetchone()
             if (row != None):
                 course_view_option = {}
                 course_view_option['show_resources_accessed'] = row[0] and 'show_resources_accessed' not in settings.VIEWS_DISABLED
+                course_view_option['show_assignment_planning_v1'] = row[1] and 'show_assignment_planning' not in settings.VIEWS_DISABLED
```

Comment:
This utility function doesn't appear to be used anywhere, but there is a bug here: I'm pretty sure you need to bump the row indices for `show_assignment_planning` and `show_grade_distribution` to `2` and `3`. Also, it would make sense to type cast each row value to to a boolean (e.g. `bool(row[0])`), since it seems to be returning 0 when it's false, while it's returning `True` the whole expression evaluates to true.